parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveStringifier.java

```diff
@@ -421,4 +422,30 @@ private String stringifyWithScale(BigInteger i) {
       }
     };
   }
+
+  static final PrimitiveStringifier UUID_STRINGIFIER = new PrimitiveStringifier("UUID_STRINGIFIER") {
+    private final char[] digit = "0123456789abcdef".toCharArray();
+    @Override
+    public String stringify(Binary value) {
+      byte[] bytes = value.getBytesUnsafe();
```

Comment:
Do you want to make 'bytes' final since you are calling getBytesUnsafe()?

Reply:
`final` would protect the reference only and not the values of the array. Making a reference final in a local scope is usually required in situations where it is accessed from e.g. lambda closures.

Reply:
Sounds good!